jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildConfiguration.java

```diff
@@ -58,6 +61,8 @@
     @Nullable private Path baseImageLayersCacheDirectory;
     private boolean allowInsecureRegistries = false;
     private ImmutableList<LayerConfiguration> layerConfigurations = ImmutableList.of();
+    private Function<AbsoluteUnixPath, Instant> fileTimestampProvider =
+        ignored -> Instant.ofEpochMilli(1000);
```

Comment:
Wonder if this should be an explicit provider implementation? I can't remember who mentioned this, but it was that the default was somewhere in code rather than set in the api? But maybe it doesn't matter so much if it's configurable (and documented).